rudder-core/src/main/scala/com/normation/rudder/services/policies/write/Cf3PolicyDraft.scala

```diff
@@ -240,16 +243,19 @@ case class ParameterEntry(
 
 /*
  * Escape string to be CFEngine compliant
- * a \ witll be escaped to \\
+ * a \ will be escaped to \\
  * a " will be escaped to \"
  * The parameter may be null (for legacy reason), and it should be checked
  */
 object ParameterEntry {
-  def escapeString(x: String) : String = {
+  def escapeString(x: String, agentType: AgentType) : String = {
     if (x == null)
       x
     else
-      x.replaceAll("""\\""", """\\\\""").replaceAll(""""""", """\\"""")
+      agentType match {
+      case AgentType.Dsc => x.replaceAll("`", "``").replaceAll("\"", "`\"")
```

Comment:
can you correct the indentation ? :)

Reply:
why is it \" that we escape, rather than simply " ?

Reply:
" is escaped because it's a double quote string, so to contains a " it needs to be escaped